lldb/source/Commands/CommandObjectProcess.cpp

```diff
@@ -1303,13 +1301,12 @@ class CommandObjectProcessSaveCore : public CommandObjectParsed {
       if (command.GetArgumentCount() == 1) {
         FileSpec output_file(command.GetArgumentAtIndex(0));
         FileSystem::Instance().Resolve(output_file);
-        SaveCoreStyle corefile_style = m_options.m_requested_save_core_style;
-        Status error =
-            PluginManager::SaveCore(process_sp, output_file, corefile_style,
-                                    m_options.m_requested_plugin_name);
+        auto core_dump_options = m_options.m_core_dump_options;
```

Comment:
Don't make a copy:
```
auto &core_options = m_options.m_core_dump_options;
```
If you use "auto" without specifying `&` it will make a copy of the settings.